### PR TITLE
Open group poll-blocking

### PR DESCRIPTION
**Pull request by RyanRory**

### CODE REVIEW

File: SessionMessagingKit/Open Groups/OpenGroupManagerV2.swift [hunk 1]
```diff
@@ -107,6 +120,20 @@ public final class OpenGroupManagerV2 : NSObject {
         Storage.shared.removeV2OpenGroup(for: thread.uniqueId!, using: transaction)
     }
     
+    // MARK: Block
+    private var blocklist: [String] = {
+        let path = Bundle.main.path(forResource: "blocklist", ofType: "txt")!
+        let content = try! String(contentsOfFile: path)
```
Comment (mpretty-cyro) [inline SessionMessagingKit/Open Groups/OpenGroupManagerV2.swift:126]: Do we want the force-unwrap and force-try here? (not sure if we want it to crash if the file is removed or becomes invalid - especially if we end up automating the updating of the file)

File: SessionMessagingKit/Open Groups/blocklist.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+fd65dd2b888a492b6e5dd2d064ab8b01871c06f67d7bdea3775b30d445f5f4a0
```
Comment (mpretty-cyro) [inline SessionMessagingKit/Open Groups/blocklist.txt:1]: Looks like this already needs to be updated 😅

**Review by mpretty-cyro — CHANGES_REQUESTED** [review]

File: SessionMessagingKit/Open Groups/OpenGroupManagerV2.swift (hunk 1 above)
Comment (RyanRory) [reply SessionMessagingKit/Open Groups/OpenGroupManagerV2.swift:126]: True, I'll change it. We don't want any crash.

File: SessionMessagingKit/Open Groups/blocklist.txt (hunk 2 above)
Comment (RyanRory) [reply SessionMessagingKit/Open Groups/blocklist.txt:1]: Yeah, there is another hash for test and I didn't include that into this PR...

**Review by mpretty-cyro — APPROVED** [review]

**Comment by RyanRory** [comment]
This PR is remained to be discussed, so close it for now.

**Pull request closed without merging**